apps/test/unit/templates/lessonOverview/LessonOverviewTest.js

```diff
@@ -176,4 +179,28 @@ describe('LessonOverview', () => {
     const wrapper = shallow(<LessonOverview {...newDefaultProps} />);
     assert.equal(wrapper.find('#unit-test-introduced-code').length, 0);
   });
+
+  it('renders standards header when standards are present', () => {
+    const standards = [
+      {
+        frameworkName: 'ngss',
+        parentCategoryShortcode: 'ESS',
+        parentCategoryDescription: 'Earth and Space Science',
+        categoryShortcode: 'ESS1',
+        categoryDescription: "Earth's Place in the Universe",
+        shortcode: '1-ESS1-1',
+        description:
+          'Use observations of the sun, moon, and stars to describe patterns that can be predicted.'
+      }
+    ];
+    const lesson = {
+      ...defaultProps.lesson,
+      standards: standards
+    };
+    const wrapper = shallow(
+      <LessonOverview {...defaultProps} lesson={lesson} />
+    );
+
+    expect(wrapper.containsMatchingElement(<h2>Standards</h2>)).to.be.true;
```

Comment:
Is it worth testing the contents of the standards section, either in this test or in a separate test file for `LessonStandards`?

Reply:
yes, apologies... I will add LessonStandardsTest.js and a .story.jsx.

Reply:
@bethanyaconnor please take another look!